modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -31,14 +31,16 @@
           :button="more.button"
           :menu="more.menu"
           @item-clicked="$emit('item-clicked', item)"
+          menu-placement="bottom-start"
+          menu-offset="40, 10"
```

Comment:
Is this legal in v-tooltip? I'm only seeing reference to using single values.

Reply:
I don't think the documentation specifies exactly but the source treats it that way (and it does works)